python/tvm/script/tir/scope_handler.py

```diff
@@ -560,10 +572,19 @@ class ThreadBinding(ForScopeHandler):
     def __init__(self):
         def thread_binding(
             begin: PrimExpr,
-            end: PrimExpr,
-            thread: str,
+            end: PrimExpr = None,
+            thread: str = None,
             annotations: Optional[Mapping[str, Object]] = None,
         ):
+            if not thread:
```

Comment:
You need to explicit check `thread is not None` (same for `end`), see the CI error